pkg/issuer/acme/http/internal/istio/istio.go

```diff
@@ -0,0 +1,74 @@
+/*
+Copyright 2021 The cert-manager Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+// By utilising this file, no dependance on `istio.io/client-go` is
+// required. Used in combination with a dynamic client, it is possible
+// to create a client for these CRDs without adding a client-go dependency.
```

Comment:
See https://github.com/istio/api/issues/1959 

Maybe expand on this comment with a GitHub link to the exact sha sum of the code you copied.
Also how about putting this in `... internal/istio/api` so that it can simply be copied from https://github.com/istio/client-go

Reply:
I added a link to the exact lines and commit.

Reply:
Great.